src/components/VHeader/VContentSwitcher/VContentSwitcherButton.vue

```diff
@@ -64,12 +75,28 @@ export default {
       () => isMinScreenMd.value || !isHeaderScrolled.value
     )
 
+    const localePath = (item, query) =>
+      app.localePath({ path: `/search/${item === 'all' ? '' : item}/`, query })
+
+    // TODO (obulat): How do we mock useRoute()?
```

Comment:
Does it work if you mock the `$route` object, similar to mocking the `$i18n` object for `useContext`?

Reply:
I removed route from here because having a link inside a button doesn't make sense